recipe/meta.yaml

```diff
@@ -1,37 +1,31 @@
-{% set version = "0.15.1" %}
+{% set version = "0.16.0" %}
 
 package:
   name: cartopy
   version: {{ version }}
 
 source:
   url: https://github.com/SciTools/cartopy/archive/v{{ version }}.tar.gz
-  sha256: c2221eaf8cb3827b026a398374f3b2e9431aa77fdf95754229232ef5236221ed
+  sha256: 688606f2a9782f14997d57142fbec89d4e8f10cd3dcb4c7c3b0d0a2dd7098cc8
   patches:
     # Cartopy geos_c patch.
     - cartopy.win.patch  # [win]
-    # Library changes taken from https://github.com/SciTools/cartopy/pull/773
-    - matplotlib2.patch
```

Comment:
@dopplershift I think it is safe to drop this patch now that https://github.com/SciTools/cartopy/pull/773 was merged.

Reply:
We should remove the file as well then?

Reply:
Yep. Forgot about that one!